docs/components/raycaster.md

```diff
@@ -58,6 +58,7 @@ AFRAME.registerComponent('collider-check', {
 | origin    | Vector3 coordinate of where the ray should originate from relative to the entity's origin.                                                            | 0, 0, 0       |
 | recursive | Checks all children of objects if set. Else only checks intersections with root objects.                                                              | true          |
 | showLine  | Whether or not to display the raycaster visually with the [line component][line].                                                                                         | false          |
+| worldCoordinates  | Determine if origin and direction are given in local or world coordinates.                                                                                         | false          |
```

Comment:
`Whether the raycaster origin and direction properties are specified in world space.`